backend/inspirehep/records/api/literature.py

```diff
@@ -327,7 +327,6 @@ def add_file(
             )
         return {
             "key": new_key,
-            "original_url": original_url or url,
```

Comment:
so you never populate `original_url`? not sure I understand.

Reply:
no, we leave the ```original_url``` as it is. We don't change the ```original_url``` to be the ```url``` as this wrong - we would be storing afs and ```/api/files..``` in it